api/definition/swagger.yaml

```diff
@@ -0,0 +1,273 @@
+            and geaographical areas requested
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerErrror"
+      parameters:
+        - $ref: "#/components/parameters/indicator_ids"
+        - $ref: "#/components/parameters/area_codes"
+      description: Get health data for indicators. Supply one or more unique indicator
+        ids in the query string. Optionally filter the results by supplying one
+        or more area codes and one or more years in the query string.
+      operationId: getHealthDataForAnIndicator
+components:
+  schemas:
+    IndicatorSummary:
+      type: object
+      description: A summary of a public health indicator
+      required:
+        - id
+        - title
+      properties:
+        id:
+          type: integer
+          format: int32
+          example: 3456
+          description: The unique identifier of the indicator
+        title:
+          type: string
+          example: "Hypertension: QOF prevalence (all ages)"
+          description: The title of the indicator
+    Indicator:
+      type: object
+      description: A public health indicator
+      required:
+        - id
+        - title
+        - definition
+      properties:
+        id:
+          type: integer
+          format: int32
+          example: 3456
+          description: The unique identifier of the indicator
+        title:
+          type: string
+          example: "Hypertension: QOF prevalence (all ages)"
+          description: The title of the indicator
+        definition:
+          type: string
+          example: "The percentage of patients with established hypertension, as
+              recorded on practice disease registers (proportion of total list
+              size)"
+          description: The definition of the indicator
+    HealthDataPoint:
+      type: object
+      description: Represents a health data point for a public health indicator with a
+        count, value, upper confidence interval, lower confidence interval and a
+        year.
+      required:
+        - year
+        - count
+        - value
+        - lowerCi
+        - upperCi
+      properties:
+        year:
+          type: integer
+          format: int32
+          example: 2023
+          description: The year that the data point is for
+        count:
+          type: number
+          format: int32
+          example: 222
+          description: The count
+        value:
+          type: number
+          format: int32
+          example: 506.60912
+          description: The value
+        lowerCi:
+          type: number
+          format: int32
+          example: 441.69151
+          description: The lower confidence interval
+        upperCi:
+          type: number
+          format: int32
+          example: 578.32766
+          description: The upper confidence interval
```

Comment:
Should these values all be defined as integers? At least some of the examples given aren't whole numbers, and we presumably can't specify that they will be for all indicators.